tests/virt_autotest/virt_utils.pm

```diff
@@ -62,6 +63,10 @@ sub repl_repo_in_sourcefile {
 sub repl_module_in_sourcefile {
     my $version = get_version_for_daily_build_guest;
     $version =~ s/fcs/sp0/;
+    my $release = ($version =~ /(\d+)-/m);
+    #We only support sle product, and only products >= sle15 has module link
+    return unless (get_required_var('DISTRI') eq 'sle' && $release >= 15);
```

Comment:
Can you use `is_sle('15+')` for clarity instead?

Reply:
is_sle does not work for us, because we have host upgrade project which will switch host version from VERSION to the real version it is installed to. We actually have tests which need to install host to be sle12sp3/sle15/sle11sp4 when it is sle12sp4 medium.

Reply:
OK, I see, but still you can replace `get_required_var('DISTRI') eq 'sle'` with `is_sle` (and keeping the rest of the line), right?

Reply:
oh, yes